src/main.js

```diff
@@ -370,4 +375,5 @@ module.exports = {
   upsertLibraries,
   buildTestSuite,
   testAndPublish,
+  colorize,
```

Comment:
the `colorize` function wasn't exported before, how come it is exported now?

Reply:
It was exported when I wanted to test something. It shouldn't be.